action/lib/Database.js

```diff
@@ -58,7 +58,16 @@ module.exports = function(dbURL, dbName) {
         return new Promise((resolve, reject) => {
             this.db.insert(params, (err, result) => {
                 if(err) {
-                    reject(err);
+                    if(err.statusCode && err.statusCode === 409) {
+                        this.getTrigger(params.triggerName)
+                        .then(() => this.disableTrigger(params.triggerName))
```

Comment:
I don't understand why all this async stuff is going on when the entire action functionality is blocking anyway...